alphadia/numba/fragments.py

```diff
@@ -159,7 +159,7 @@ def impl(inst, slices):
 
             precursor = np.arange(len(slices), dtype=np.uint32)
 
-            for i, (start_idx, stop_idx, step) in enumerate(slices):
```

Comment:
What is the idea of this rule? that it's marked if it's unused?

Reply:
yes, that's part of this ruleset